pkgs/development/tools/jq/default.nix

```diff
@@ -59,6 +56,12 @@ stdenv.mkDerivation rec {
   # jq is linked to libjq:
   ++ lib.optional (!stdenv.isDarwin) "LDFLAGS=-Wl,-rpath,\\\${libdir}";
 
+  postFixup = ''
+    # Break the dependency cycle: $dev refers to $bin via propagated-build-outputs, and
+    # $bin refers to $dev because of https://github.com/jqlang/jq/commit/583e4a27188a
```

Comment:
```suggestion
    # $bin refers to $dev because of https://github.com/jqlang/jq/commit/583e4a27188a2db097dd043dd203b9c106bba100
```

Reply:
Do you fear someone might forge a commit hash starting with `583e4a27188a` in the future? ;)

Reply:
It is bad practice to use short git hashes in anything that is long lived even before there where tools to forge hashes.